js/main.js

```diff
@@ -26,20 +34,24 @@ var $exitButton = document.querySelector('.exit-btn');
 var $backLinkFeatured = document.querySelector('.back-arrow-feat');
 var $backLinkDetail = document.querySelector('.back-arrow-detail');
 var $topLinkDetail = document.querySelector('.up-arrow');
-var $bookmarkIcon = document.querySelector('.bookmarks');
-var $bookmarkAction = document.querySelector('.bookmark-action');
-var $searchIcon = document.querySelector('.search');
-var $form = document.querySelector('form');
-var $input = document.querySelector('input');
-var $suggestions = document.querySelector('.suggestions');
 var $featuredGames = document.querySelector('.featured-games');
 
 function getData(url, task) {
   var xhr = new XMLHttpRequest();
   xhr.open('GET', url);
   xhr.responseType = 'json';
-  xhr.addEventListener('load', function (event) { task(xhr.response); });
-  xhr.send();
+  xhr.addEventListener('load', function (event) {
+    task(xhr.response);
+    $loadView.hidden = true;
+  });
+
+  if (navigator.onLine) {
+    xhr.send();
+  } else {
+    $errorView.hidden = false;
+  }
+
+  $loadView.hidden = false;
```

Comment:
Instead of using the `hidden` property, I would use a class you can add and remove to hide and show the element.

Reply:
You are doing this in other places in your code too, please address those as well.